webtau-graphql/src/test/resources/test-schema.graphql

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright 2020 webtau maintainers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+type Task {
+  id: String
```

Comment:
in graphql, you'd typically define this as `id: ID!` or even `taskId: ID!`
Shouldn't impact the outcome of any tests as ID's tend to be strings.

Reply:
Yup, indeed, oversight on my part.